src/fork-stream.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {PassThrough, Readable, Transform} from 'stream';
+import File = require('vinyl');
+
+export function forkStream(stream: NodeJS.ReadableStream) {
```

Comment:
Consider annotating the return value as a NodeJS.ReadableStream because it's not intended that the receiver write to it.